### PR TITLE
Add tests and coverage.

### PR DESCRIPTION
**Pull request by perry**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module.exports = function(config) {
+  config.set({
+    basePath: '',
+    frameworks: ['jasmine'],
+    files: [
+        'app/bower_components/lodash/lodash.js',
```
Comment (rogerhutchings) [inline karma.conf.js:6]: `basePath: './app'`

...then you can remove `app` from file paths

File: package.json [hunk 2]
```diff
@@ -32,7 +33,13 @@
     "gulp-uglify": "^1.1.0",
     "gulp-usemin": "^0.3.11",
     "gulp-watch": "^4.1.1",
+    "jasmine-core": "^2.2.0",
```
Comment (rogerhutchings) [inline package.json]: Pin all the versions?

Comment (perry) [inline package.json]: Good idea

File: karma.conf.js (hunk 1 above)
Comment (perry) [inline karma.conf.js:6]: Should probably have `bowerPath = basePath + /bower_components` as well.

Comment (perry) [inline karma.conf.js:6]: Ooooh, I see what you mean!

Comment (rogerhutchings) [inline karma.conf.js:6]: Yeah, it's just a convenience setting really :)

Comment (perry) [inline karma.conf.js:6]: Hmm, this broke coverage.

**Pull request merged**